common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -147,6 +204,56 @@ def _publish_event(self, event_name, result, **kwargs):
         self.runtime.publish(self, "edx.librarycontentblock.content.{}".format(event_name), event_data)
         self._last_event_result_count = len(result)  # pylint: disable=attribute-defined-outside-init
 
+    @classmethod
+    def publish_selected_children_events(cls, block_keys, format_block_keys, publish_event):
+        """
+        Publish selected events on child objects.  # REVIEW: Useless. Rewrite.
```

Comment:
Maybe, I'm missing a convention, but does this comment just mean this needs to go through docs or is this just a TODO?